# AI Researcher & Speech Processing
### _Current Position - Ph.D. Student in Nagoya Unvi.

#### About: 
Ph.D. in Plant Breeding and Genetics with 6 years of research experience in genetics, genomics, and data analysis of crop plants. Skilled in experimental design, statistical analysis, handling large genomic data and development and deployment of R/Shiny packages for plant breeding applications. Interested in a research or industry role involving genetics, plant breeding, bioinformatics, or Artificial Intelligence for plant breeding. 

<!-- #### Publication Statistics:
Cumulative Impact Factor: **34.171**

Total Citations: **39** <a href="https://scholar.google.com/citations?user=KnuQm0cAAAAJ&hl=en" target="_blank">Google Scholar</a> 
-->

#### Technical Skills:
- Programming Languages: _Proficient in R, Python, Linux command line; experience with Shiny, Google Colab._
- Phenomics and Genomics: _Expertise in molecular breeding techniques, NGS data analysis, QTL/GWAS, genomic selection, designing field trials, managing phenotyping pipelines, and analyzing high-throughput phenotypic and genotypic data._
- Data Science: _Multivariate analysis, machine learning, large scale genomic data analysis and visualization._
  
## üéì Education
- **Ph.D.**, Genetics and Plant Breeding | <a href="https://tnau.ac.in/" target="_blank">TamilNadu Agricultural University</a> (_FEB 2021 - Thesis submitted on 31st OCT 2023_)		  		
- **M.Sc.**, Genetics and Plant Breeding	| <a href="https://tnau.ac.in/" target="_blank">TamilNadu Agricultural University</a> (_AUG 2017 - DEC 2020_)	 			        		
- **B.Sc.**, Agriculture | <a href="https://cattheni.edu.in/" target="_blank">College of Agricultural Technology</a> (_JUL 2013 - JUL 2017_)

## üî¨ Research Experience
**Ph.D. research @<a href="https://tnau.ac.in/" target="_blank">TNAU</a> (_2021 - 2023_)**
- Developed and applied advanced tissue culture techniques like embryo culture to rapidly generate **somaclonal variants of rice** with improved salinity stress tolerance.
- Identified and validated novel genomic regions **(QTLs)** governing salinity tolerance through **molecular mapping** in rice F<sub>2</sub> populations along with thorough phenotypic and biochemical characterization.
- Employed **multi-omics approaches** including GC-MS metabolomics to uncover key genes and metabolites linked to superior osmotic/ionic adjustment under salt stress.

**Research scholar @ <a href="https://www.icrisat.org/" target="_blank">ICRISAT</a> (_2018 - 2020_)**
- Single plant phenotying of diverse germplasm accessions (Sorghum, Peral millet, Pigeon pea ‚Äì genotyped 1,980 plants/samples combined) to examine intra and inter accession genetic diversity. 
- Single plant genotyping of accessions using **<a href="https://www.diversityarrays.com/" target="_blank">DArTSeq</a> based SNPs**.
- Harnessed whole genome genotyping, simulations, and predictive modeling to provide integrated insights into the dynamics of genetic variation across a species' range, by quantifying genomic diversity within and between isolated subpopulations, and developed optimized statistical frameworks to guide sustainable sampling regimes that limit genetic drift.

## üé§ Workshop and Conferences
- Attended the workshop on "<a href="https://twitter.com/GenebankICRISAT/status/1169137453472669696" target="_blank">A15 DArTSeq Data Analysis</a>" at <a href="https://www.cimmyt.org/" target="_blank">CIMMYT</a>, Mexico
- Attended the international conference on <a href="https://twitter.com/GenebankICRISAT/status/1158297753371308032" target="_blank">"Neglected and Underutilized Crop Species for Food, Nutrition, Energy and Environment"</a> at NIPGR, New Delhi. Awarded a travel grant in recognition of contribution. 

## ‚úçÔ∏è Articles & Blogs
- <a href="https://medium.com/@victorallan" target="_blank">Medium Articles</a>

## üìú Publications

1. Allan Victor, Mani Vetriventhan, Ramachandran Senthil, S. Geetha, Santosh Deshpande, Abhishek Rathore, Vinod Kumar, Prabhat Singh, Surender Reddymalla, and V√¢nia CR Azevedo. **"Genome-wide DArTSeq genotyping and phenotypic based assessment of within and among accessions diversity and effective sample size in the diverse sorghum, pearl millet, and pigeonpea landraces."**_Frontiers in Plant Science_ 11 (2020): 587426. (DOI: <a href="https://doi.org/10.3389/fpls.2020.587426" target="_blank">https://doi.org/10.3389/fpls.2020.587426</a>)
2. Allan, V. (2023) **‚ÄòPB-Perfect: A Comprehensive R-Based Tool for Plant Breeding Data Analysis‚Äô**, PB - Perfect. Available at: <a href="https://allanbiotools.shinyapps.io/pbperfect/" target="_blank">https://allanbiotools.shinyapps.io/pbperfect/</a>.


## üìû Contact

| **Name**   | Jingyi FENG | 
| **Gmail**   | <a href="mailto:fuuseii1222@gmail.com">fuuseii1222@gmail.com</a> | 
| **LinkedIn**   | <a href="https://www.linkedin.com/in/jingyi-feng-59a9a8242/" target="_blank">Visit Page</a> | 
<!--
iiynjgef/iiynjgef is a ‚ú® special ‚ú® repository because its `README.md` (this file) appears on your GitHub profile.
You can click the Preview link to take a look at your changes.

page reference: https://github.com/Allan-gitrepos/My_resume
-->
